packages/millicast-sdk/src/Publish.js

```diff
@@ -236,32 +245,33 @@ export default class Publish extends BaseWebRTC {
     promises = await Promise.all([getLocalSDPPromise, signalingConnectPromise])
     const localSdp = promises[0]
 
-    const workerBlob = new Blob([workerString])
-    const workerURL = URL.createObjectURL(workerBlob)
-    if (this.worker) {
-      this.worker.terminate()
```

Comment:
Shouldn't you keep the cleanup of the worker in case it was set previously?

Reply:
This is the same as in the View comment, I believe

Reply:
Yes same